rusk-wallet/src/bin/command/history.rs

```diff
@@ -156,14 +158,54 @@ pub(crate) async fn transaction_from_notes(
                     && th.height == decoded_note.block_height
             });
 
-            // If outgoing txs found, this should be the change or any
-            // other output created by the tx result
-            // (like withdraw or unstake)
             if let Some(th) = outgoing_tx {
-                th.amount += note_amount
-
-                // If no outgoing txs found, this note should belong to a
-                // preconfigured genesis state
+                // Outgoing txs found, this should be the change or any
+                // other output created by the tx result
+                // (like withdraw or unstake)
+                th.amount += note_amount;
+            } else {
+                // No outgoing txs found, this note should either belong to a
+                // preconfigured genesis state or is the result of a
+                // moonlight to phoenix conversion.
+                let moonlight_tx_events = gql
+                    .moonlight_history_at_block(
+                        public_address,
+                        decoded_note.block_height,
+                    )
+                    .await?;
+                let Some(moonlight_history) =
+                    moonlight_tx_events.full_moonlight_history
+                else {
+                    continue;
```

Comment:
Correct me if I'm wrong, but it seems that if we don't find a transaction that created the current note we simply continue here?
If so I think this is not correct. If we have a note for which we cannot trace the origin, our history is incorrect and we want to error.

Reply:
It could belong to a preconfigured genesis state.

Reply:
Can we double check that they are genesis notes? And are they shown somehow?

Reply:
I've made a separate issue for this here https://github.com/dusk-network/rusk/issues/3736.